chsdi/tests/integration/__init__.py

```diff
@@ -32,6 +35,19 @@ def tearDown(self):
         del self.testapp
         del self.grids
 
+    @contextmanager
+    def getSession(self):
+        session = scoped_session(sessionmaker())
+        yield session
+        session.close()
+
+    def getRandomFeatureId(self, bodId):
+        models = models_from_bodid(bodId)
+        with self.getSession() as session:
+            query = session.query(models[0]).limit(1)
+            reslt = query.one()
```

Comment:
Sure I can do it, we need to change all our tests where we have hardcoded ids in the integration tests in the future as well. Let's make it 500, build is long enough already.